kovri.md

```diff
@@ -1,64 +1,68 @@
-Kovri Lightweight C++ I2P Router Script
-
-Version 1 - sgp
-Version 2 - scoobs
-Version 3 - sgp
+The Kovri Script
 
 Until this point, this video series has covered how Monero obfuscates information stored on the blockchain.
 Ring signatures obscure the sender.
-Stealth addressing prevents anyone but the sender and the recipient from knowing the recipient owns Monero.
-RingCT hides the amount of Monero transmitted.
-However, some personally identifiable information can still be leaked in the process of sending Monero.
-This video will focus on a technology that closes that gap, called Kovri.
+Stealth addressing prevents anyone, other than the sender and the recipient, from knowing that the recipient owns Monero.
+RingCT hides the amount of Monero that's transmitted.
+However, some personally identifiable information will still be leaked in the process of sending Monero.
+This video focuses on the technology that closes this gap of privacy. This technology is called Kovri.
 
 Suppose Alice wants to send a transaction to Bob.
 She generates a transaction on her local device, and gets ready to broadcast it to the network, so it can be included in the next block.
-Alice is connected to only a small portion of the nodes on the Monero network. Her transaction request will need to reach as many nodes as possible - ideally all of them - for her to have the greatest chance of this transaction getting included in the next block.
-Let's say she is connected to five other nodes, so she broadcasts her transaction to all five.
-These five nodes then forward her transaction to all the nodes they are connected to, and so on, until many (or all) nodes have this transaction.
+Alice is connected to only a small portion of the nodes on the Monero network. Her transaction request will need to reach as many nodes as possible - ideally all of them - for her to have the greatest chance of this transaction to be included in the next block.
+Let's say she connects to five other nodes. We'll call each node "Bob". Once connected, she broadcasts her transaction to all five Bobs.
+These Bobs then forward her transaction to all the nodes they are connected to, and so on, until many (or all) nodes have this transaction.
 
-Unfortunately, Alice takes a risk to broadcast her transaction.
-Some of the nodes on the network may be logging the IP addresses that they get transaction requests from.
+Unfortunately, Alice takes a risk when broadcasting her transaction.
+Some of the nodes on the network may be logging the IP addresses they get when transaction requests are given.
 An IP address is like your home address for your internet connection.
```

Comment:
"An IP address is like a home address for the internet."

Reply:
Agreed, good enough.